Source/Core/Core/ConfigManager.cpp

```diff
@@ -900,7 +900,12 @@ bool SConfig::AutoSetup(EBootBS2 _BootBS2)
 
       m_region = ContentLoader.GetRegion();
       const char* retrieved_region_dir = GetDirectoryForRegion(m_region);
-      set_region_dir = retrieved_region_dir ? retrieved_region_dir : EUR_DIR;
+      if (!retrieved_region_dir)
```

Comment:
You may want to move the region setting code to its own function so this is all kept together if any refactoring or future changes happen.